src/cpp/session/modules/SessionDataImportV2.R

```diff
@@ -387,7 +390,7 @@
             paramsPackage = "readr",
             options = options,
             optionTypes = optionTypes,
-            cacheFileExtension = ".txt"
+            cacheFileExtension = c(".txt")
```

Comment:
Shouldn't be necessary -- `".txt"` is already a vector of length one (cf. `identical(".txt", c(".txt"))`)